umbral/pre.py

```diff
@@ -482,10 +486,14 @@ def decrypt(capsule: Capsule, priv_key: UmbralPrivateKey,
         bob_priv_key = priv_key
         key = _open_capsule(capsule, bob_priv_key, alice_pub_key)
         dem = UmbralDEM(key)
-        cleartext = dem.decrypt(ciphertext)
+
+        bytes_original_capsule = capsule._original_to_bytes()
+        cleartext = dem.decrypt(ciphertext, authenticated_data=bytes_original_capsule)
     else:
         key = _decapsulate_original(priv_key.bn_key, capsule)
         dem = UmbralDEM(key)
-        cleartext = dem.decrypt(ciphertext)
+
+        bytes_capsule = bytes(capsule)
```

Comment:
Usually, we've been saying `<entity_type>_bytes` to name a bytes representation of something (ie, `capsule_bytes`, not `bytes_capsule`)